src/openforms/appointments/tests/test_confirmation_emails.py

```diff
@@ -127,7 +128,7 @@ def test_send_confirmation_mail_disabled(self):
             contact_details={"lastName": "Powers", "email": "austin@powers.net"},
         )
 
-        send_confirmation_email(appointment.submission)
+        schedule_emails(appointment.submission.id)
```

Comment:
the behaviour has changed I believe, the previous test call would actually result in mails being scheduled if the logic was wrong, but now even if the logic is wrong I believe the test/assertion will pass because we don't send mail synchronously inside `schedule_emails`. For test robustiness/correctness, I think this needs wrapping in `CELERY_TASK_ALWAYS_EAGER=True` to for synchronous processing *or* you need to assert that none of the mail sending functions are called/scheduled.